### ISSUE TITLE
engadget

### ISSUE THREAD

**Issue by liujunhui2** [gfwlist/gfwlist#2278]
engadget is part of Yahoo, and Yahoo is stopping serving to China
*.engadget.com is preferred

**Comment by cicku** [comment]
`chinese.engadget.com` has been in the list for many years.

Are you saying that the apex domain is also blocked?

**Comment by liujunhui74** [comment]
Yes, the apex domain. It's not blocked, just that Yahoo has stopped serving to China.

**Comment by zdm9981** [comment]
https://github.com/gfwlist/gfwlist/issues/2096#issuecomment-1224793346
